go.mod

```diff
@@ -9,14 +9,14 @@ require (
 	github.com/aws/aws-sdk-go-v2/config v1.1.1
 	github.com/aws/aws-sdk-go-v2/credentials v1.1.1
 	github.com/aws/aws-sdk-go-v2/service/route53 v1.1.1
-	github.com/btcsuite/btcd/btcec/v2 v2.2.0
 	github.com/cespare/cp v0.1.0
 	github.com/cloudflare/cloudflare-go v0.14.0
 	github.com/cockroachdb/pebble v0.0.0-20230209160836-829675f94811
 	github.com/consensys/gnark-crypto v0.10.0
 	github.com/crate-crypto/go-kzg-4844 v0.3.0
 	github.com/davecgh/go-spew v1.1.1
 	github.com/deckarep/golang-set/v2 v2.1.0
+	github.com/decred/dcrd/dcrec/secp256k1/v4 v4.2.0
```

Comment:
Since btcec is also using a version of [v4.0.1]( https://github.com/btcsuite/btcd/blob/master/go.mod#L12), I recommend keeping 4.0.1 here

Reply:
I don't think so. the 4.2.0 doesn't have any breaking changes.

Reply:
If you just want to replace the underlying btcec package in this package, please keep the underlying package version the same. And If you also want to upgrade a package, this should be in another PR.